<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupSettingsWidget</class>
 <widget class="QWidget" name="BackupSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="unconfiguredPlaceholderPage">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,0,0,3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="unconfiguredPlaceholderImageLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>128</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5,3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="unconfiguredPlaceholderLabel">
           <property name="text">
            <string notr="true">&lt;To enable backup add more drives to use them as backup storage in Storage Management&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="offlinePlaceholderPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="offlinePlaceholderLabel">
         <property name="text">
          <string>Server is offline. Backup settings are not available.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsPage">
      <layout class="QVBoxLayout" name="settingsPageLayout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="nx::vms::client::desktop::BackupStatusWidget" name="backupStatusWidget" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="tabBarHolder" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabBar" name="tabBar" native="true"/>
          </item>
          <item>
           <widget class="QTabBar" name="emptyTabBarFiller" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="settingsStackedWidget">
         <widget class="QWidget" name="viewWidgetPage">
          <layout class="QVBoxLayout" name="viewWidgetPageLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="nx::vms::client::desktop::MessageBar" name="messageBar" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bandwidthWidgetPage">
          <layout class="QVBoxLayout" name="bandwidthWidgetPageLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="nx::vms::client::desktop::BackupBandwidthSettingsWidget" name="backupBandwidthSettingsWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::BackupStatusWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/server/widgets/backup_status_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header location="global">QtWidgets/QTabBar</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::BackupBandwidthSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/server/widgets/backup_bandwidth_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::MessageBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
